.travis.yml

```diff
@@ -36,9 +39,14 @@ install:
       sudo apt-get install mpich
     fi
 
+  # upgrade numpy to make sure `import tensorflow` does not cause numpy version error
+  # RuntimeError: module compiled against API version 0xb but this version of numpy is 0xa
```

Comment:
moving it to before_install section might be better, there is already a pip upgrade check as well